lib/go-tc/stats_summary.go

```diff
@@ -187,3 +187,115 @@ type StatsSummaryLastUpdatedAPIResponse struct {
 	Response StatsSummaryLastUpdated `json:"response"`
 	Alerts
 }
+
+// StatsSummaryV5 is an alias for the latest minor version for the major version 5.
+type StatsSummaryV5 StatsSummaryV51
+
+// StatsSummaryV51 is a summary of some kind of statistic for a CDN and/or
+// Delivery Service.
+type StatsSummaryV51 struct {
```

Comment:
Shouldn't this struct be name `StatsSummaryV50`, since this is for api 5.0, and not 5.1?

Reply:
Fixed it.